.github/actions/setup-helm-docs/action.yml

```diff
@@ -4,11 +4,13 @@ inputs:
   version:
     description: 'Version of helm-docs'
     required: false
-    default: 1.13.1
```

Comment:
Not sure I understand why using default input param is a problem

Reply:
when you have a workflow which has an optional input invoking this actionoptional input, you would hope it will use the default value but instead will be evaluated to empty string

Reply:
it is not applied when called from another action/workflow where the version is an input and has an empty default, so either I repeat the same default on each or I do the check for empty

it did break here https://github.com/Alfresco/alfresco-process-infrastructure-deployment/actions/runs/9566428280/job/26406144776?pr=2214#step:5:120